### DISCUSSION TITLE
Upgrade to v3.2.0

### DISCUSSION THREAD

**Discussion by porscheme** [vesoft-inc/nebula#4470] [Q&A]
Hi @wey-gu 

How do I upgrade our current v3.1.0 Nebula Cluster to v3.2.0?

- As per v3.2.0 upgrade [docs](https://docs.nebula-graph.io/3.2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-from-300-to-latest/), we need to stop all Nebula Graph services. But how do I stop all services?

**Comment by wey-gu** [comment]
You should refer to https://docs.nebula-graph.io/3.2.0/nebula-operator/9.upgrade-nebula-cluster/ instead as you are using the k8s deployment, where the things needed are changing versions in CRD, and the operator will do everything else for you comparing to baremetal deployment as https://docs.nebula-graph.io/3.2.0/4.deployment-and-installation/3.upgrade-nebula-graph/upgrade-nebula-from-300-to-latest/

**Reply by wey-gu** [reply]
And that's the sweet part of K8s/ Operator design pattern I think :)
